index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const app = express();
+const { urlencoded } = require('express');
+const bodyParser = require('body-parser');
+
+const { userRouter } = require('./routers/user');
+const { categoryRouter } = require('./routers/category');
+const { brandRouter } = require('./routers/item-brand');
+const { workerRouter } = require('./routers/workers');
+
+app.use(express.json());
+app.use(urlencoded({
+    extended: true,
+}));
+app.use(bodyParser.json({
+    type: 'application/json',
+}));
```

Comment:
express.json() and bodyParser.json you need for parse JSON incoming data? Then why u add both? U can leave only express.json

Reply:
Done